internal/app/init/main.go

```diff
@@ -287,54 +306,60 @@ func startKubernetesServices(data *userdata.UserData) {
 	)
 }
 
+func reboot() {
+	// See http://man7.org/linux/man-pages/man2/reboot.2.html.
+	unix.Sync()
```

Comment:
do we want to do sync() on any restart? I think it makes sense, idk if that might lead to code hanging on shutdown? we can add some timer here

Reply:
added timer there